src/compiler/types.ts

```diff
@@ -2782,6 +2788,19 @@ namespace ts {
         ArrayLiteralOrCallOrNewExcludes = ContainsSpreadElementExpression,
     }
 
+    /* @internal */
+    export const enum NodeEmitFlags {
+        EmitEmitHelpers = 1 << 0,           // Any emit helpers should be written to this node.
+        EmitExportStar = 1 << 1,            // The export * helper should be written to this node.
+        EmitSuperHelper = 1 << 2,           // Emit the basic _super helper for async methods.
+        EmitAdvancedSuperHelper = 1 << 3,   // Emit the advanced _super helper for async methods.
+        UMDDefine = 1 << 4,                 // This node should be replaced with the UMD define helper.
+        NoLexicalEnvironment = 1 << 5,      // A new LexicalEnvironment should *not* be introduced when emitting this node.
+        SingleLine = 1 << 6,                // The contents of this node should be emit on a single line.
+        MultiLine = 1 << 7,                 // The contents of this node should be emit on multiple lines.

```

Comment:
Is the idea that if neither single nor multiple is specified, then the output is "unspecified" and could be either?